### PR TITLE
Add config for OIDC login

### PR DESCRIPTION
**Pull request by SilviaAmAm**
Partially fixes https://github.com/maykinmedia/open-archiefbeheer/issues/390

Screenshot from 2024-10-02 16-33-42

### CODE REVIEW

File: src/templates/login/login.tsx [hunk 1]
```diff
@@ -40,6 +45,22 @@ export const LoginTemplate: React.FC<LoginTemplateProps> = ({
         defaultMessage: "inloggen",
       });
 
+  const _labelOidcLogin = labelOidcLogin
+    ? labelOidcLogin
+    : intl.formatMessage({
+        id: "mykn.templates.Login.labelOidcLogin",
+        description: "templates.Login: The label for the OIDC login button.",
+        defaultMessage: "OIDC Login",
```
Comment (svenvandescheur) [inline src/templates/login/login.tsx:53]: Maybe improve the (Dutch) default label. I prefer something short but at least somewhat B1, e.g. ("Inloggen met organisatie account"). 

Don't forget to create the translations files:

> Messages can be extracted by running npm run makemessages after which they end up in src/lib/i18n/messages, they can be compiled after translation using npm run compilemessages.

File: src/templates/login/login.tsx [hunk 2]
```diff
@@ -51,7 +72,11 @@ export const LoginTemplate: React.FC<LoginTemplateProps> = ({
         <Body>
           {slotLogo || CustomLogo || <Logo />}
           <Hr />
-          <Form labelSubmit={ucFirst(_labelLogin)} {...formProps} />
+          <Form
+            labelSubmit={ucFirst(_labelLogin)}
+            secondaryActions={secondaryActions}
+            {...formProps}
```
Comment (svenvandescheur) [inline src/templates/login/login.tsx:78]: I'm wondering if it should be possible to overrule the default login and only allow OIDC?

**Review by svenvandescheur — CHANGES_REQUESTED** [review]

File: src/templates/login/login.tsx (hunk 2 above)
Comment (SilviaAmAm) [reply src/templates/login/login.tsx:78]: I don't know! But could be useful :thinking:

File: src/lib/i18n/compiled/en.json [hunk 3]
```diff
@@ -37,5 +37,6 @@
   "mykn.components.Select.labelClear": "clear value",
   "mykn.components.Sidebar.collapse": "Collapse sidebar",
   "mykn.components.Sidebar.expand": "Expand sidebar",
-  "mykn.templates.Login.labelLogin": "inloggen"
+  "mykn.templates.Login.labelLogin": "inloggen",
```
Comment (svenvandescheur) [inline src/lib/i18n/compiled/en.json:40]: Whoops, that one was already untranslated, can you fix it :)

**Pull request merged**